users/moults31/vscode.c

```diff
@@ -0,0 +1,90 @@
+/* Copyright 2021 moults31
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "vscode.h"
+
+bool process_record_vsc(uint16_t keycode, keyrecord_t *record) {
+    bool rv = true;
+
+    switch (keycode) {
+        case M_VSC_TERMFOCUS:
+            if (record->event.pressed) {
+                SEND_STRING(SS_LCTRL("`"));
```

Comment:
Just a heads up, most of these could be done with simple modded keycodes. 

Eg `C(KC_GRV)`, or `C(S(KC_2))`.   And the `tap_code16` function will support these, too.

Reply:
Thanks for the heads-up, I'll definitely be going back through and cleaning these up sooner than later